test/hotspot/jtreg/serviceability/jvmti/vthread/BreakpointInYieldTest/libBreakpointInYieldTest.cpp

```diff
@@ -303,11 +308,9 @@ Java_BreakpointInYieldTest_check(JNIEnv *jni, jclass cls) {
   err = jvmti->SetEventNotificationMode(JVMTI_DISABLE, JVMTI_EVENT_VIRTUAL_THREAD_START, NULL);
   check_jvmti_status(jni, err, "check: error in JVMTI SetEventNotificationMode: disable VIRTUAL_THREAD_START");
 
-  err = jvmti->SetEventNotificationMode(JVMTI_DISABLE, EXT_EVENT_VIRTUAL_THREAD_MOUNT, NULL);
-  check_jvmti_status(jni, err, "check: error in JVMTI SetEventNotificationMode: disable VIRTUAL_THREAD_MOUNT");
+  set_ext_event_notification_mode(jvmti, jni, JVMTI_DISABLE, "VirtualThreadMount", NULL);
```

Comment:
I think this should only replace the call to `SetEventNotificationMode()`, leaving the use of `check_jvmti_status()` alone.

Reply:
Updated.